actions/lib/action.py

```diff
@@ -9,6 +9,7 @@ def run(self, **kwargs):
 
     def __init__(self, config):
         super(JenkinsBaseAction, self).__init__(config)
+        self.config = config
```

Comment:
Unnecessary, given [this line in st2](https://github.com/StackStorm/st2/blob/master/st2common/st2common/runners/base_action.py#L44).

```python
class Action(object):
    # ...
    def __init__(self, config=None, action_service=None):
        """
        :param config: Action config.
        :type config: ``dict``
        ...
        """
        self.config = config or {}
 ```

Reply:
PyCharm complained that it should be defined in the init part :) I just wanted to make that warning go away.

Reply:
You might want to pass `*args` and `**kwargs` to `super(JenkinsBaseAction, self).__init__`, so it doesn't break if we change this API.

Reply:
And yes, please remove it. If you need an example of a pack's base action, check out the [st2 pack's base action](https://github.com/StackStorm-Exchange/stackstorm-st2/blob/master/actions/lib/action.py).

Reply:
Removed!

Reply:
For history's sake: I decided that since the st2 base action doesn't pass in `*args` or `**kwargs`, that we will need to fix this across all packs at once.